.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+env:
+  global:
+    - firebase_bucket=gs://tbatv-prod-hrd.appspot.com
+
+branches:
+  only:
+  - master
+
+before_install:
+  - openssl aes-256-cbc -K ${ENCRYPTED_KEY_NAME} -iv ${ENCRYPTED_IV_NAME} -in gcp-service-key.json.enc -out gcp-service-key.json -d
+  - nvm install 9
+
+install:
+  - travis_retry npm install
+  - travis_retry npm install -g react-native-cli
+
+script:
+  - react-native bundle --platform ios --dev false --assets-dest ios --entry-file index.ios.js --bundle-output ios/main.jsbundle --reset-cache
```

Comment:
I know this is probably my poor documentation but - for this repo on CI, let's go ahead and remove that `--reset-cache`

Reply:
Removed in 7fce4e570876ccd42476b29daeca5c0a06d8ef3a